### PR TITLE
submitting, patching, listings form

### PR DESCRIPTION
**Pull request by mohdabed123**
### Description
submitting listing form

### CODE REVIEW

File: src/components/create-listing/CreateListingForm/create-listing-form-schema.js [hunk 1]
```diff
@@ -43,7 +44,7 @@ const listingFormSchema = yup
     yearBuilt: positiveIntegerValidator.max(new Date().getFullYear()),
   })
   .test('atLeastOneCheckBoxIsSelected', null, (obj) => {
-    if (obj.identifier > 0) {
+    if (obj.isAgent === true || obj.isOwner === true) {
```
Comment (AhmadAlShwaiki) [inline src/components/create-listing/CreateListingForm/create-listing-form-schema.js:47]:
```suggestion
    if (obj.isAgent || obj.isOwner) {
```

**Review by AhmadAlShwaiki — APPROVED** [review]

File: src/pages/Listings/ListingPage.jsx [hunk 2]
```diff
@@ -6,18 +6,19 @@ import OfferDetails from 'components/listing-page/OfferDetails';
 import Overview from 'components/listing-page/Overview';
 import SimilarListings from 'components/listing-page/SimilarListings';
 import ListingPageSlider from 'components/listing-page/Slider';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useParams } from 'react-router';
```
Comment (AseelRousan-op) [inline src/pages/Listings/ListingPage.jsx:10]:
```suggestion
import { useParams } from 'react-router-dom';
```

File: src/components/create-listing/FormInputFields/index.jsx [hunk 3]
```diff
@@ -256,9 +277,13 @@ function FormInputFields({
           label="Overview"
           name="overview"
           register={register}
-          limit={overviewCharsLimit - overviewText.length}
-          value={overviewText}
-          onChange={updateOverviewText}
+          value={values.overview}
+          // eslint-disable-next-line no-unsafe-optional-chaining
```
Comment (AseelRousan-op) [inline src/components/create-listing/FormInputFields/index.jsx:281]: Can we add a rule in the `.eslintrc` file to disable this error:
```
"no-unsafe-optional-chaining": "off",
```

```suggestion
```

**Review by AseelRousan-op — APPROVED** [review]
two minor comments, Looks good otherwise

**Review by AseelRousan-op — APPROVED** [review]
Thank you for addressing them 👍

**Pull request merged**